frontend/src/audio.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+import Alpine from 'alpinejs';
+import './player';
+
+document.addEventListener('alpine:initialized', () => {
+  let article = document.querySelector('#audio-page');
+  /** type @{HTMLElement} */ // JSDoc types
+  if (article instanceof HTMLElement) {
+    window.dispatchEvent(
+      new CustomEvent('archive:toggle-play', {
+        detail: {
+          fileId: article.dataset.audioNumber,
+          shouldPlay: false,
+          contentDetails: {
+            title: article.dataset.audioTitle,
+            dateForHumans: article.dataset.dateforhumans,
+            dateUncertain: article.dataset.dateuncertain,
+            location: article.dataset.location,
+            locationUncertain: article.dataset.locationuncertain,
```

Comment:
Why don't you use camel casing for these properties? I see you successfully use them for the other ones.

Reply:
Updated camelCases for properties.